### PR TITLE
Build bundle dependencies in separate docker volume [SCI-1199]

### PR DESCRIPTION
**Pull request by okriuchykhin**
https://biosistemika.atlassian.net/browse/SCI-1199

you'll need docker engine version 1.12.0+ and docker-compose with version 1.6.0+

### CODE REVIEW

**Comment by mz3944** [comment]
It doesn't work for these versions:
Docker engine: 1.12.3
Docker compose: 1.7.0

File: docker-compose.yml [hunk 1]
```diff
@@ -1,22 +1,35 @@
+version: '2.1'
```
Comment (mz3944) [inline docker-compose.yml]: Change to '2.0'.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,22 +1,35 @@
...
+    container_name: scinote_web_development
+    stdin_open: true
+    tty: true
+    depends_on:
+      - db
+    ports:
+      - "3000:3000"
+    command: >
+      bash -c "rm -f /usr/src/app/tmp/pids/server.pid &&
+               (bundle check || bundle install) &&
```
Comment (mz3944) [inline docker-compose.yml]: Add `rake deface:precompile &&`.

**Comment by okriuchykhin** [comment]
Changed version of compose file and added target to Makefile for compiling deface.

**Comment by mz3944** [comment]
Nice! Merge away! :1st_place_medal:

**Pull request merged**